src/marker.rs

```diff
@@ -680,6 +696,55 @@ impl MarkerExpression {
             (MarkerValue::QuotedString(l_string), MarkerOperator::Equal, MarkerValue::Extra) => {
                 extras.contains(l_string)
             }
+            (
+                MarkerValue::MarkerEnvVersion(MarkerValueVersion::PythonVersion),
+                operator,
+                MarkerValue::QuotedString(r_string),
+            ) => {
+                // ignore all errors block
+                (|| {
+                    // The right hand side is allowed to contain a star, e.g. `python_version == '3.*'`
+                    let (r_version, r_star) = Version::from_str_star(r_string).ok()?;
+                    let operator = operator.to_pep440_operator()?;
+                    // operator and right hand side make the specifier
+                    let specifier = VersionSpecifier::new(operator, r_version, r_star).ok()?;
+
+                    // We're passing a lot of versions here, e.g. 3.8 .. 3.100, but since we're
```

Comment:
Is it guaranteed that these are passed in order? The API doesn't enforce it or is it just best practice?

Reply:
tbh the answer it "doesn't really matter", i don't even see that function call in the flamegraph